launch_template.tf

```diff
@@ -14,6 +14,13 @@ resource "aws_launch_template" "sensor_launch_template" {
     }
   }
 
+  metadata_options {
+    http_endpoint               = "enabled"
+    http_tokens                 = "required"
+    http_put_response_hop_limit = 1
```

Comment:
This setting may prevent containers (exporters, etc) from using an instance role to connect to S3.

(I know there was an issue in the past where this needed to be >= 2 to use an instance role to connect to S3, unsure whether there was some other fix for that.)